src/main/java/ladder/domain/Person.java

```diff
@@ -0,0 +1,39 @@
+package ladder.domain;
+
+import java.util.Objects;
+
+public class Person {
+
+    private final static int MAX_NAME_LENGTH = 5;
+    private final static String MAX_NAME_ERROR_MESSAGE = "이름은 5글자를 넘을 수 없습니다.";
```

Comment:
- 상수 선언 시, 키워드를 `static final` 순으로 작성해보시면 좋을 것 같아요.
- 숫자 리터럴을 하드 코딩하는 대신에, 위에 선언된 상수를 활용해보시면 좋을 것 같아요.
```suggestion
    private static final int MAX_NAME_LENGTH = 5;
    private static final String MAX_NAME_ERROR_MESSAGE = "이름은 " + MAX_NAME_LENGTH + "글자를 넘을 수 없습니다.";
```